### PR TITLE
Measure and set request time and _request_details documentation

### PR DESCRIPTION
**Pull request by machielvdw**
### What are the changes and their implications?
Calculates response time of requests using Date.now(). The response time is then added to the `request` object as a new property `responseTime`.

### TODO: 
- [x] Documentation
 
## Checklist

- [x] Pull request is made to the "develop" branch
- [x] Tests added
- [x] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

**Comment by Gervwyk** [comment]
Todo: Add `_request_details` operator to docs.

**Comment by machielvdw** [comment]
> Todo: Add `_request_details` operator to docs.

f784eacaede6af485cb40d887fd0de608075d2d3

File: packages/docs/operators/_request_details.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    pageId: _request_details
+    pageTitle: _request_details
+    filePath: operators/_request_details.yaml
+    env: Client Only
+    types: |
+      ```
+      (requestId: string): any
+      ```
+    description: |
+      The `_request_details` operator returns detailed information of a request. If the request has not yet been called, or is still executing, the returned value is `null`.
```
Comment (SamTolmay) [inline packages/docs/operators/_request_details.yaml:14]: Does it return `null` if the request is executing?

Comment (machielvdw) [reply packages/docs/operators/_request_details.yaml]: Thanks for picking that up, no it doesn't. I have amended here

**Review by Gervwyk — DISMISSED** [review]

**Review by SamTolmay — APPROVED** [review]

**Pull request merged**